src/services/dex.js

```diff
@@ -34,922 +34,571 @@ const assetUTXOsToIgnore = {};
 const contractUTXOsReservedFor = {};
 
 export default {
-  fetchMarkets() {
-    return new Promise((resolve, reject) => {
-      try {
-        const currentNetwork = network.getSelectedNetwork();
-        axios.get(`${currentNetwork.aph}/markets?contractScriptHash=${assets.DEX_SCRIPT_HASH}`)
-          .then((res) => {
-            resolve(res.data.markets);
-          })
-          .catch((e) => {
-            alerts.exception(`APH API Error: ${e}`);
-          });
-      } catch (e) {
-        reject(`Failed to fetch markets. ${e.message}`);
-      }
-    });
-  },
-
-  formOrderBook(asks, bids) {
-    const book = {
-      asks: [],
-      bids: [],
-    };
-
-    asks.forEach((ask) => {
-      ask.price = toBigNumber(ask[0]);
-      ask.quantity = toBigNumber(ask[1]);
-      book.asks.push(ask);
-    });
-    bids.forEach((bid) => {
-      bid.price = toBigNumber(bid[0]);
-      bid.quantity = toBigNumber(bid[1]);
-      book.bids.push(bid);
-    });
+  async checkUnspentsReservedState(assetId, unspents) {
```

Comment:
None of these changes are anything new except for the one with my comment. I simply alphabetized them.